back-office/orders/orders/Order Page View.md

```diff
@@ -516,7 +516,7 @@ The edit shipping button on each row can edit the information in a popin. You ca
 
 4.  **Merchandising returns tab**
 
-The merchandise return tab lists **the date, the tracking number, the type, Carrier (which is the return status) and the quantity**. When the merchandise return is disabled the tab displayed: "no merchandise returned yet".
+The merchandise return tab lists **the date, the tracking number, the type, the carrier (= return status), and the number (= return reference number)**. When the merchandise return is disabled the tab displayed: "no merchandise returned yet".
```

Comment:
If I understand the issue well, the "tracking number" column is no longer displayed, and the "Quantity" column is replaced by the "number" column.

```suggestion
The merchandise return tab lists **the date, the type, the carrier (= return status), and the number (= return reference number)**. When the merchandise return is disabled the tab displayed: "no merchandise returned yet".
```

Reply:
Actually, there is no quantity column, I don't know why it was written in the specifications. @MatShir, any idea? As for the tracking number, it turned out to be the return ID, cf. https://github.com/PrestaShop/PrestaShop/issues/19200#issuecomment-638101058.

Reply:
@MatShir, fixed in this PR, I let you correct it for the order page specifications.

Reply:
I can confirm, it is the ID. It is a mistake from me when I started the specs.